how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -1,15 +1,18 @@
 # How to use Kata Containers and CRI (containerd plugin) with Kubernetes
 
 This document describes how to set up a single-machine Kubernetes cluster. 
-The Kubernetes cluster will use the CRI containerd plugin and Kata Containers to launch untrusted workloads.
+The Kubernetes cluster will use the [CRI containerd plugin](https://github.com/containerd/cri) and [Kata Containers](https://katacontainers.io) to launch untrusted workloads.
```

Comment:
We've tended to specify https://github.com/kata-containers as the project homepage for docs on github. Note that that URL does contain a link to https://katacontainers.io but the former is going to be quicker to navigate to find the code I think.